tests/test_double_blind_conference.py

```diff
@@ -1625,7 +1625,7 @@ def test_withdraw_submission(self, client, test_client, helpers):
             forum = notes[0].forum,
             replyto = notes[0].forum,
             readers = ['everyone'],
-            writers = [conference.id + '/Paper{number}/Authors'.format(number = notes[0].number), conference.id],
+            writers = [conference.id],
```

Comment:
This should match the invitation which is defined as:
`[ conference.get_id(), conference.get_program_chairs_id() ]`